controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -0,0 +1,171 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2016-2019 Intel Corporation
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "wfa_ca.h"
+
+#include <beerocks/bcl/beerocks_string_utils.h>
+#include <beerocks/bcl/network/network_utils.h>
+
+#include <beerocks/tlvf/beerocks_message.h>
+#include <beerocks/tlvf/beerocks_message_bml.h>
+
+#include <easylogging++.h>
+
+#include <algorithm>
+
+using namespace beerocks;
+
+std::string wfa_ca::wfa_ca_status(eWfaCaStatus status)
+{
+    switch (status) {
+    case RUNNING:
+        return "RUNNING";
+    case INVALID:
+        return "INVALID";
+    case ERROR:
+        return "ERROR";
+    case COMPLETE:
+        return "COMPLETE";
+    default:
+        break;
+    }
+    return std::string();
+}
+
+wfa_ca::eWfaCaCommand wfa_ca::wfa_ca_command(std::string command)
```

Comment:
argument should be const if possible

Reply:
continueing the previous comment, the function name should be `wfa_ca_command_from_string()` like we have in other areas in our codebase.

Reply:
I have chosen to not pass it as const reference since I am changing it inside the function (transform).
Changed it to wfa_ca_command_from_string() though.